### DISCUSSION TITLE
Precise Style Transfer

### DISCUSSION THREAD

**Discussion by cubiq** [cubiq/ComfyUI_IPAdapter_plus#607] [Announcements]
I added a new weight type called "style transfer precise". It offers less bleeding between the style and composition layers. It can be useful when the reference image is very different from the image you want to generate.

Attached a few examples of standard vs precise

| ![precise](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/0603779b-b061-4e1b-912f-9545f6db89db) | ![precise2](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/a7d69b03-20bc-4610-9bcd-92b04297ad4b) |
|---|---|
| ![precise3](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/d438ab33-07da-4eb8-8826-8d1f533299f7) | ![precise4](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/7a72e92a-7763-4b4c-9ce4-542f1d51fdb6) |

**Comment by apluka34** [comment]
Thanks

**Comment by cubiq** [comment]
posted a video about it, in the coming days I'll make a dedicated node just for precise style transfer

https://www.youtube.com/watch?v=ewKM7uCRPUg

cc @xiaohu2015

**Comment by edtjulien** [comment]
Thank you for that! It's mind blowing.

Do you have a suggestion of weights and layers for 1.5 which reduce the bleeding? In the video, it's for SDXL only right?

**Comment by xhoxye** [comment]
Why is the result different from yours?
So I need to fill in the value?

![QQ截图20240719201356](https://github.com/user-attachments/assets/8443b2a3-fad3-45ec-883d-3920786a781d)

**Comment by dxhark** [comment]
Hello author, I am an industrial designer and a heavy user of iPad adapters. I would like to collaborate with you on training. How can I obtain your contact information
